packages/dd-trace/src/config.js

```diff
@@ -93,7 +103,12 @@ function propagationStyle (key, option, defaultValue) {
 
 class Config {
   constructor (options) {
-    options = options || {}
+    options = this.options = options || {}
+    options = this.options = {
+      ...options,
+      appsec: options.appsec != null ? options.appsec : options.experimental?.appsec,
+      iastOptions: options?.experimental?.iast
```

Comment:
unnecessary optional chaining
```suggestion
      iastOptions: options.experimental?.iast
```